1.1/spec/17-Annex-B.adoc

```diff
@@ -162,11 +162,10 @@ From GeoSPARQL 1.0 use, the most commonly observed use of a `Geometry` is in rel
     a geo:Geometry ;
     skos:prefLabel "Geometry Y";
     geo:asWKT "MULTIPOLYGON (((149.06016 -35.23610, 149.060620 -35.236043, ... , 149.06016 -35.23610)))"^^geo:wktLiteral ;
-    geo:inSRS <http://www.opengis.net/def/crs/EPSG/0/4326> ;
 .
 ```
 
-Here the `Geometry` instance has data in WKT form and declares an SRS, WGS84: using the EPSG identifier.
+Here the `Geometry` instance has data in WKT form and, since no SRS is declared, WGS84 is the assumed, default, SRS.
```

Comment:
"WGS84" should be "CRS84". And using CRS instead of SRS would be more precise.

Reply:
Fixed as mostly requested in commit https://github.com/opengeospatial/ogc-geosparql/pull/162/commits/e8fae58e396d5a1c9762cc55929e3ca0cfa17f9e. I left the reference to "WGS84", not CRS84 as the full name is WGS84 but it's clearly indicated to be a CRS by the sentence context.